index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Kyle Hartman's Portfolio</title>
+    <style>
+        body {background-color: rgb(8, 5, 28);}
+        h1 {color: rgb(175, 152, 20);}
+        p {color: rgb(23, 96, 64);}
+    </style>
+</head>
+
+<body>
+    <h1>KYLE HARTMAN</h1>
+    <section id="About">
```

Comment:
While this id will still work starting with a capital letter, if you are curious about learning standard conventions for as you move forward with the project, you'll want to make ids `inCamelCaseLikeThis` and classes `in-kabob-case-like-this`.